packages/halo-theme/src/custom-elements/ef-combo-box.less

```diff
@@ -17,6 +17,13 @@
     height: 100%;
     width: @button-height;
   }
+  // TODO: fix unfinished style of clear button
+  [part~='button-clear']:hover {
+    color: @input-focused-border-color;
+  }
+  [part~='button']:active {
+    color: @button-hover-text-color;
```

Comment:
`:active` state of the clear button is invisible.

Reply:
Fixed.